src/assets/locales/fr-CH.json

```diff
@@ -33,7 +33,6 @@
     "title": "Contrôle des présences",
     "no-lessons": "Aucune leçon n'a lieu à la date sélectionnée.",
     "no-lesson-presences": "Aucun-e élève n'a été sélectionné-e.",
-    "previously-absent": "Absent-e la leçon précédente",
```

Comment:
Fehlt in der französischen Übersetzung noch der Eintrag für `preceding-absence`? Der ist zur Zeit nur in der deutschen.